src/index.ts

```diff
@@ -1,12 +1,30 @@
 const privateData = new WeakMap()
 
+const observer = new IntersectionObserver(entries => {
+  for(const entry of entries) {
+    if (entry.isIntersecting) {
+      const {target} = entry 
+      observer.unobserve(target)
+      if (!(target instanceof IncludeFragmentElement)) return
+      if (target.loading === 'lazy') {
+        handleData(target)
+      }
+    }
+  }
+}, {
+  rootMargin: '0px 0px 256px 0px',
+  threshold: 0.01
```

Comment:
Could we include your comment about these numbers here in the code? The rationale being that if I was reading this code this would look like magic numbers to me and I'd have to go hunting for the PR to see the justifications for them.